src/test/ui/duplicate/duplicate-check-macro-exports.rs

```diff
@@ -1,6 +1,6 @@
 pub use std::panic;
 
-#[macro_export]
-macro_rules! panic { () => {} } //~ ERROR the name `panic` is defined multiple times
+#[macro_export] //~ ERROR the name `panic` is defined multiple times
+macro_rules! panic { () => {} }
```

Comment:
Is there a version of this test with the export in a submodule?

Reply:
It looks like there is not, I'll add this test case.